packages/gatsby/src/commands/build-html.ts

```diff
@@ -22,7 +25,10 @@ const runWebpack = compilerConfig =>
     })
   })
 
-const doBuildRenderer = async (program, webpackConfig) => {
+const doBuildRenderer = async (
+  program: IProgram,
+  webpackConfig: webpack.Configuration
+): Promise<string> => {
```

Comment:
As a rule, are we going to always declare the return type even when it can be inferred?

Reply:
Yeah I think so. Easier to visually grok and we have it set as a lint rule